src/main/java/games/strategy/engine/persistence/serializable/ProxyFactoryRegistryFactory.java

```diff
@@ -0,0 +1,24 @@
+package games.strategy.engine.persistence.serializable;
+
+import games.strategy.internal.persistence.serializable.PropertyBagMementoProxy;
+import games.strategy.internal.persistence.serializable.VersionProxy;
+import games.strategy.persistence.serializable.ProxyFactoryRegistry;
+
+/**
+ * Factory for creating pre-configured instances of {@link ProxyFactoryRegistry}.
```

Comment:
Heh, I'll have to remember "kingdom of nouns." :smile:  Yes, I was not happy with the "factory factory" name either.  The other naming convention I considered was `ProxyFactoryRegistries#platform()`, which effectively returned a singleton rather than creating a new instance (there really should only ever be one created anyway).

I don't think this logic should be moved to `ProxyFactoryRegistry`.  The `g.s.persistence.serializable` package provides general functionality related to building a persistence layer, and shouldn't really have any knowledge of the individual proxies required for a specific domain model.

IIRC, during the original design review, I indicated I was not happy with placing this "pre-configured" instance in the predecessor to this class.  I think I might have asked if `ClientContext` would be a better location for this instance (I just checked my notes and I have a TODO for moving it to `ClientContext`).  **Do you think `ClientContext` would be more appropriate?**